docs/index.md

```diff
@@ -69,71 +80,55 @@ The easiest way to get started with Swarm is to use the
 $ docker pull swarm
 ```
 
-## Set up Swarm nodes
+## Set up Swarm
 
-Each Swarm node will run a Swarm node agent. The agent registers the referenced
-Docker daemon, monitors it, and updates the discovery backend with the node's status.
+1. Log into **each machine** and do the following:
 
-The following example uses the Docker Hub based `token` discovery service:
-
-1. Create a Swarm cluster using the `docker` command.
+    Start the Docker daemon with the `-H` flag. This ensures that the Docker remote API
+    is available over TCP to the *Swarm Manager*.
 
     ```bash
-    $ docker run --rm swarm create
-    6856663cdefdec325839a4b7e1de38e8 # <- this is your unique <cluster_id>
+	$ docker -H tcp://0.0.0.0:2375 -d
     ```
 
-    The `create` command returns a unique cluster ID (`cluster_id`). You'll need
-    this ID when starting the Swarm agent on a node.
-
-2. Log into **each node** and do the following.
-
-    1. Start the Docker daemon with the `-H` flag. This ensures that the Docker remote API on *Swarm Agents* is available over TCP for the *Swarm Manager*.
-
-        ```bash
-    		$ docker -H tcp://0.0.0.0:2375 -d
-        ```
+2. Start the Swarm manager on any machine (even on your laptop).
 
-    2. Register the Swarm agents to the discovery service. The node's IP must be accessible from the Swarm Manager. Use the following command and replace with the proper `node_ip` and `cluster_id` to start an agent:
+    The simplest way to set up the Swarm manager is to provide a comma-separated
+    list of machine addresses:
```

Comment:
Perhaps make a mention that, although using a static list of IP-addresses is the *easiest* way, in most cases it's not the recommended way. Add that a Discovery service is more flexible and "to set up a Discovery service, see [link to discovery section]"